From 7ca02b08c6f5bd8fdff3276881cf862e6103c955 Mon Sep 17 00:00:00 2001
From: Xie xiaobo <X.Xie@freescale.com>
Date: Thu, 30 Apr 2009 15:02:43 +0800
Subject: [PATCH] ENGR00112000 MX25: Remove warning messages for build in system.c

Compile arch/arm/mach-mx25/system.c will output warning messages.
This patch fixed that.

Signed-off-by: Xie Xiaobo <X.Xie@freescale.com>
---
 arch/arm/mach-mx25/system.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx25/system.c b/arch/arm/mach-mx25/system.c
index 654348e..203fcd9 100644
--- a/arch/arm/mach-mx25/system.c
+++ b/arch/arm/mach-mx25/system.c
@@ -121,7 +121,6 @@ void mxc_cpu_lp_set(enum mxc_cpu_pwr_mode mode)
 
 	__raw_writel(reg, MXC_CCM_CCTL);
 }
-EXPORT_SYMBOL(mxc_cpu_lp_set);
 
 /*!
  * This function puts the CPU into idle mode. It is called by default_idle()
-- 
1.5.4.4

